install_config/topics/configuring_for_vsphere_registry.adoc

```diff
@@ -0,0 +1,98 @@
+// Module included in the following assemblies:
+//
+// * install_config/configuring_for_vsphere.adoc
+
+[id='vsphere-configuring-registry_{context}']
+= Configuring the {product-title} registry for vSphere
+The steps below define the steps define the manual process to 
```

Comment:
The following steps define the manual process of storage creation, which is used . . .